src/demultiplex/main.nf

```diff
@@ -126,6 +134,11 @@ workflow run_wf {
           "output_multiqc": "output_multiqc"
         ]
       )
+      | map{ id, state -> 
+        def newId = "${date}/".toString()
```

Comment:
Does this not fail when we have multiple runs that are processed at the same time?